app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1602,44 +1602,53 @@ table li .account_details {
   margin-bottom: 50px;
 }
 
-.row-items {
-  max-width: 1000px;
+// start non-admin items menu
+.items table {
+  background-color: #303030;
+  margin-top: 50px;
+  border: 1px solid #303030;
 }
 
-.items {
-  border: 1px solid red;
-  margin: 10px auto;
+.items td {
+  background-color: #303030;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.